api/src/main/webapp/WEB-INF/web.xml

```diff
@@ -3,4 +3,9 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd">
     <display-name>AllOfUs Workbench API</display-name>
+
+    <error-page>
+        <error-code>503</error-code>
```

Comment:
Should add a clause for 500s as well, e.g. go to https://all-of-us-workbench-test.appspot.com/ with the network tab open and look at the `/workspaces` response.

Reply:
I removed this. See PR comments for more info.